k8s/pkg/templates/operation/dask-repartition.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     spec:
       containers:
       - name: dask-repartition-operation
-        image: localhost:5001/dask-repartition
+        image: ghcr.io/v6d-io/v6d/dask-repartition
```

Comment:
Why ghcr instead of local registry?

Reply:
The local registry is only for testing, for others using vineyard operator in their environment, the image should be able to be downloaded normally.